### ISSUE TITLE
Missing support for ScenarioOutlines

### ISSUE THREAD

**Issue by UFOMelkor** [vsiakka/gherkin-lint#39]
When running gherkin-lint for features that contains ScenarioOutlines I get results like this:
```
  1     Feature file does not have any Scenarios     no-files-without-scenarios
  10    Unknown gherkin node type ScenarioOutline    indentation
  22    Unknown gherkin node type ScenarioOutline    indentation
  34    Unknown gherkin node type ScenarioOutline    indentation
```

**Comment by timbru31** [comment]
@UFOMelkor noticed the same, working on a fix - a regression of 1.0.x 😞

**Comment by vsiakka** [comment]
@UFOMelkor, @timbru31 fixed it with v1.0.2
